src/features/unified-share-modal/ContactRestrictionNotice.js

```diff
@@ -19,6 +19,22 @@ import type { contactType as Contact } from './flowTypes';
 
 import './ContactRestrictionNotice.scss';
 
+const SINGLE_CONTACT = 'singleContact';
+const MULTIPLE_CONTACTS = 'multipleContacts';
+const RESTRICTION_JUSTIFICATION_ENABLED = 'restrictionJustificationEnabled';
+const RESTRICTION_JUSTIFICATION_DISABLED = 'restrictionJustificationDisabled';
+
+const restrictionNoticeMessageMap = {
```

Comment:
Keeping these on the component's file for now since they are not relevant outside the component.